citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java

```diff
@@ -3003,7 +3035,7 @@ public void doFinish ( RunData data)
 			// Remove session collection
 			state.removeAttribute(STATE_CITATION_COLLECTION_ID);
 			state.removeAttribute(STATE_CITATION_COLLECTION);
-	
+		
 			state.removeAttribute("fromListPage");
```

Comment:
It just makes the code harder to patch on Sakai as it's more likely that the line you've cleaned up will have been changed in Sakai and so cause a conflict when applying.